cmd/deviceservice/rm/rm.go

```diff
@@ -35,20 +35,15 @@ func NewCommand() *cobra.Command {
 			     "deviceservice rm --name <name>",
 		Short: "Removes device service by name or ID",
 		Long:  `Removes a device service from the core-metadata DB.`,
-		RunE: func(cmd *cobra.Command, args []string) (err error){
+		Run: func(cmd *cobra.Command, args []string) {
```

Comment:
thought you said RunE is the way to go .. why did you reverse here?

Reply:
Run functions have been returned back to RunE.